README.md

```diff
@@ -1 +1,154 @@
-# ros-frontend
\ No newline at end of file
+# Resource Optimization Service (Frontend)
+
+Resource Optimization is a service aimed to provide existing Red Hat Enterprise Linux customers with an ability to grasp control over their investment in the Public Cloud. Read more about it [here](https://www.redhat.com/en/blog/optimize-public-cloud-workloads-rhel-red-hat-insights-resource-optimization)
+
+# Prerequisites
+
+### Install
+
+- [Node](https://nodejs.org/en/download/)
+- [Docker](https://docs.docker.com/get-docker/)
+
+
+# Setup Development Environment
+
+1. To get started with Resource Optimization Service (ROS), make sure you meet the prerequisites mentioned in both ros frontend and backend repos
+
+2. Clone all required repositories in one folder:
+
+- git clone https://github.com/RedHatInsights/insights-chrome.git
+- git clone https://github.com/RedHatInsights/insights-proxy.git
+- git clone https://github.com/RedHatInsights/ros-frontend.git
+- git clone https://github.com/RedHatInsights/ros-backend.git
```

Comment:
As you are giving reference of ros backend repo, will it make sense to swap between these two steps.
ie. first clone repositories and then follow pre-requisites for both ros-frontend & ros-backend repositories.